pkg/sensor/queue/scaler_test.go

```diff
@@ -64,11 +63,9 @@ func (s *scalerTestSuite) TestScaleSize() {
 
 	for name, c := range cases {
 		s.Run(name, func() {
-			err := os.Setenv("ROX_MEMLIMIT", strconv.Itoa(c.sensorMemLimit))
```

Comment:
I think we should have a linter that detects usage of `os.Setenv` in a test

Reply:
```
git grep os.Setenv | grep _test.go
pkg/env/setting_test.go:	a.NoError(os.Setenv(name, "foobar"))
pkg/env/setting_test.go:	a.NoError(os.Setenv(name, "qux"))
pkg/env/setting_test.go:	a.NoError(os.Setenv(name, ""))
pkg/env/setting_test.go:			a.NoError(os.Setenv(name, tt.value))
pkg/env/setting_test.go:	a.NoError(os.Setenv(name, "qux"))
pkg/env/setting_test.go:	a.NoError(os.Setenv(name, ""))
pkg/env/setting_test.go:	a.NoError(os.Setenv(name, "1h"))
pkg/features/features_test.go:		err := os.Setenv(feature.EnvVar(), envSetting)
pkg/features/features_test.go:				assert.NoError(t, os.Setenv(feature.EnvVar(), oldValue))
pkg/sensor/queue/scaler_test.go:			err := os.Setenv("ROX_MEMLIMIT", strconv.Itoa(c.sensorMemLimit))
pkg/sensor/queue/scaler_test.go:				err := os.Setenv("ROX_SENSOR_BUFFER_SCALE_CEILING", strconv.Itoa(c.bufferCeiling))
pkg/sensor/queue/scaler_test.go:	err := os.Setenv("ROX_MEMLIMIT", "definitelyNotAnInteger")
pkg/sensor/queue/scaler_test.go:			err := os.Setenv("ROX_MEMLIMIT", strconv.Itoa(c.sensorMemLimit))
pkg/sensor/queue/scaler_test.go:			err = os.Setenv("TEST_1", strconv.Itoa(c.setValue))
```

Reply:
oh, I wonder when it is too many linters.